.chloggen/ingest_tls_openshift.yaml

```diff
@@ -0,0 +1,27 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use TLS via OpenShift service annotation when gateway/multitenancy is disabled
+
+# One or more tracking issues related to the change
+issues: [963]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  This will use the openshift service annotation to generate TLS certs, in order to use this feature TLS on the 
+  distributor should be enabled in this way:
+  ```
+    apiVersion: tempo.grafana.com/v1alpha1
+    kind:  TempoStack
+    spec:
+        distributor:
```

Comment:
there is no `distributor` under `spec` .

Reply:
my bad, I delete the template part

Reply:
Fixed.